google-area120-tables-v1alpha1/synth.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import synthtool.languages.ruby as ruby
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICMicrogenerator()
+library = gapic.ruby_library(
+    "google/area120/tables", "v1alpha1",
+    proto_path="google/area120/tables/v1alpha1",
+    extra_proto_files=[
+        "google/cloud/common_resources.proto",
+    ],
+    generator_args={
+        "ruby-cloud-gem-name": "google-area120-tables-v1alpha1",
+        "ruby-cloud-title": "Area 120 Tables V1alpha1",
+        "ruby-cloud-description": "Using the Area120 Tables API, you can query for tables, and update/create/delete rows within tables programmatically.",
```

Comment:
Even though the module name has no separator ("Area120"), I believe the brand/product name should be "Area 120 Tables".

Reply:
Yes, you're right. Updated.